scripts/pre_commit_linter.py

```diff
@@ -376,12 +377,19 @@ def _lint_py_files(config_pylint, files_to_lint, result):
         print 'Linting Python files %s to %s...' % (
             current_batch_start_index + 1, current_batch_end_index)
 
-        try:
-            # This prints output to the console.
-            lint.Run(current_files_to_lint + [config_pylint])
-        except SystemExit as e:
-            if str(e) != '0':
-                are_there_errors = True
+        # These lines print output to the console.
+        # This line invokes Pylint.
+        pylinter = lint.Run(
+            current_files_to_lint + [config_pylint],
+            exit=False).linter
+
+        # These lines invoke Pycodestyle.
+        style_guide = pycodestyle.StyleGuide(config_file=config_pycodestyle)
+        report = style_guide.check_files(current_files_to_lint)
```

Comment:
pycodestyle_report perhaps (so that the if check below is more clearly a check for two different linting systems)

Reply:
Done.